recipes/libyuv/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "20201106":
```

Comment:
```suggestion
  "cci.20201106":
```

https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases

Reply:
fixed